GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -1164,6 +1164,8 @@ string BuildPathFilter(string? path)
                     $"--format=\"{startOfObjectId}%H\"",
                     "--name-only",
                     "--follow",
+                    "--all",
+                    "--diff-merges=separate",
```

Comment:
Can/should these be conditioned on something? E.g., add those only if necessary. Are these always required?

Reply:
Do you mean some kind of setting? Otherwise I am not really sure. I can imagine that checking _whether_ the flags would be relevant in the first place (i. e. whether there are any commits in unmerged branches and any merge commits) could take longer time than simply using them directly.

Reply:
We have a setting on "follow" already (and one for only rename). This is a next level and would be a third setting which would be confusing.

Reply:
So should we include the `--diff-merges` option under the _Follow renames in file history_ setting? The `--all` option is kind of separate and doesn't cause such a big performance regression.